Core/OxidEE_Events.php

```diff
@@ -200,46 +197,66 @@ private static function _deleteOrderTransactionTable()
     {
         $sQuery = "DROP TABLE IF EXISTS `wdoxidee_ordertransactions`";
 
-        $oDb = oxDb::getDb();
-        $oDb->Execute($sQuery);
+        self::$oDb->Execute($sQuery);
+    }
+
+    /**
+     * Add Wirecard's payment methods defined in payments.xml
+     */
+    private static function _addPaymentMethods()
+    {
+        $oLogger = Registry::getLogger();
+        $oConfig = Registry::getConfig();
+        $sShopBaseURL = $oConfig->getShopUrl();
+        $xmldata = simplexml_load_file($sShopBaseURL . "modules/wirecard/paymentgateway/payments.xml");
+        if ($xmldata) {
+            foreach ($xmldata->payment as $payment) {
+                $array = array(
+                    "OXID" => $payment->oxid
+                );
+
+                $sQuery = "INSERT INTO " . 'oxpayments' . "(`OXID`, `OXACTIVE`, `OXTOAMOUNT`, `OXDESC`, `OXDESC_1`,
```

Comment:
I would prefer to re-format this query block to make it more readable (e.g. single inserted value per line).